components/executors/dind/patches/executor.ConfigMap.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: executor-config
+# Refer to https://docs.sourcegraph.com/admin/executors/deploy_executors_binary#step-2-setup-environment-variables on how to populate these variables
+data:
+  EXECUTOR_FRONTEND_URL: "http://sourcegraph-frontend.default.svc.cluster.local:30080"
```

Comment:
If the customer doesn't deploy Sourcegraph in the `default` namespace does this need to manually be overrided via a patch?

Reply:
We can possibly use the shorthand url 🤔 

>Just crossed my mind that namespace gets globally set on every object meaning the executor manifests will have the same namespace applied as the rest of Sourcegraph.
As such only the service shorthand url should be necessary http://sourcegraph-frontend/ if I'm not mistaken